man/chage.1.xml

```diff
@@ -90,23 +97,24 @@
 	</term>
 	<listitem>
 	  <para>
-	    Set the date or number of days since 1970-01-01 on which the
-	    user's account will no longer be accessible. The date may also
-	    be expressed in the format YYYY-MM-DD.
-	    A user whose account is locked must
-	    contact the system administrator before being able to use the
-	    system again.
+	    Set the date or number of days since 1970-01-01 on which the user's
+	    password will expire and their account will no longer be accessible.
+	    The value can be specified as a date in the YYYY-MM-DD format or as
+	    a number of days since 1970-01-01.
+	    The date is interpreted using the UTC timezone.
+	    If the password expires, the user must contact the system administrator
+	    to regain access to the system.
```

Comment:
```
Set the date or number of days since 1970-01-01
on which the user's password will expire
and their account will no longer be accessible.
The value can be specified as a date in the YYYY-MM-DD format
or as a number of days since 1970-01-01.
The date is interpreted using the UTC timezone.
If the password expires,
the user must contact the system administrator
to regain access to the system.
```